rest/data/patch_intent.go

```diff
@@ -19,20 +19,14 @@ type DBPatchIntentConnector struct{}
 
 func (p *DBPatchIntentConnector) AddPatchIntent(intent patch.Intent, queue amboy.Queue) error {
 	patchDoc := intent.NewPatch()
-	projectRef, err := model.FindOneProjectRefByRepoAndBranchWithPRTesting(patchDoc.GithubPatchData.BaseOwner,
+	_, err := model.FindOneProjectRefByRepoAndBranchWithPRTesting(patchDoc.GithubPatchData.BaseOwner,
 		patchDoc.GithubPatchData.BaseRepo, patchDoc.GithubPatchData.BaseBranch)
 	if err != nil {
 		return gimlet.ErrorResponse{
 			StatusCode: http.StatusInternalServerError,
 			Message:    "failed to fetch project_ref",
 		}
 	}
-	if projectRef == nil {
-		return gimlet.ErrorResponse{
```

Comment:
I think it's still valuable to return here. For example, if the project has PR testing disabled we don't want to continue to create the patch.

Reply:
returns nil

Reply:
I think we should still log this to grip; it's possible that we will want this information if users ask why a PR isn't being made, but logging at the Debug level might be more accurate (and then returning nil).

Reply:
Oh just saw your comment that we already log at the debug level so disregard.